packit/config/job_config.py

```diff
@@ -24,7 +24,8 @@ class JobType(Enum):
     build = "build"
     sync_from_downstream = "sync_from_downstream"
     copr_build = "copr_build"
-    production_build = "production_build"  # koji build
+    production_build = "production_build"  # upstream koji build
```

Comment:
this was indeed an unfortunate naming from our side, we should do something about it long-term